frontend/src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,46 @@
+import { Field } from 'formik'
+import { SelectableOption } from '../../types/forms'
+import { Icon } from '../Icon/Icon'
+
+export type CheckboxProps = {
+	/** Provide an name to uniquely identify the Checkbox input. */
+	name: string
+	/** Provide a label to provide a description of the Checkbox input that you are exposing to the user. */
+	labelText: string
+	/** A list of options to choose from. */
+	options: SelectableOption[]
+	/** Provides assistance on how to fill out a field. */
+	helperText?: string
+	/** When set to true, a '*' symbol will be displayed next to the label, indicating that the field is required. */
+	labelRequired?: boolean
+}
+
+/**
+ * Checkbox, can only be used with Formik
+ */
+
+// MC: The <fieldset> element may not be the best choice in this context, due to its default styling.
+export const Checkbox: React.FC<CheckboxProps> = ({ name, labelText, helperText, options, labelRequired = false }) => {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/checkbox_role

> Note: The first rule of ARIA is if a native HTML element or attribute has the semantics and behavior you require, use it instead of re-purposing an element and adding ARIA. Instead use the native [HTML checkbox of <input type="checkbox">](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox) (with an associated [<label>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)), which natively provides all the functionality required:

Wenn man input und label mit for nutzt braucht man keine weiteren ARIA elemente :)